vnext/src/Libraries/Components/Button.windows.js

```diff
@@ -26,107 +27,225 @@ import type {ColorValue} from '../StyleSheet/StyleSheet';
+    converted to the uppercased form.
    */
   title: string,
 
   /**
-   * Handler to be called when the user taps the button
+    Handler to be called when the user taps the button. The first function
+    argument is an event in form of [PressEvent](pressevent).
    */
   onPress: (event?: PressEvent) => mixed,
 
   /**
-   * If true, doesn't play system sound on touch (Android Only)
-   **/
+    If `true`, doesn't play system sound on touch.
+
+    @platform android
+
+    @default false
+   */
   touchSoundDisabled?: ?boolean,
 
   /**
-   * Color of the text (iOS), or background color of the button (Android)
+    Color of the text (iOS), or background color of the button (Android).
+
+    @default {@platform android} '#2196F3'
+    @default {@platform ios} '#007AFF'
    */
   color?: ?ColorValue,
 
   /**
-   * TV preferred focus (see documentation for the View component).
+    TV preferred focus.
+
+    @platform tv
+
+    @default false
    */
   hasTVPreferredFocus?: ?boolean,
 
   /**
-   * TV next focus down (see documentation for the View component).
-   *
-   * @platform android
+    Designates the next view to receive focus when the user navigates down. See
+    the [Android documentation][android:nextFocusDown].
+
+    [android:nextFocusDown]:
+    https://developer.android.com/reference/android/view/View.html#attr_android:nextFocusDown
+
+    @platform android, tv
    */
   nextFocusDown?: ?number,
 
   /**
-   * TV next focus forward (see documentation for the View component).
-   *
-   * @platform android
+    Designates the next view to receive focus when the user navigates forward.
+    See the [Android documentation][android:nextFocusForward].
+
+    [android:nextFocusForward]:
+    https://developer.android.com/reference/android/view/View.html#attr_android:nextFocusForward
+
+    @platform android, tv
    */
   nextFocusForward?: ?number,
 
   /**
-   * TV next focus left (see documentation for the View component).
-   *
-   * @platform android
+    Designates the next view to receive focus when the user navigates left. See
+    the [Android documentation][android:nextFocusLeft].
+
+    [android:nextFocusLeft]:
+    https://developer.android.com/reference/android/view/View.html#attr_android:nextFocusLeft
+
+    @platform android, tv
    */
   nextFocusLeft?: ?number,
 
   /**
-   * TV next focus right (see documentation for the View component).
-   *
-   * @platform android
+    Designates the next view to receive focus when the user navigates right. See
+    the [Android documentation][android:nextFocusRight].
+
+    [android:nextFocusRight]:
+    https://developer.android.com/reference/android/view/View.html#attr_android:nextFocusRight
+
+    @platform android, tv
    */
   nextFocusRight?: ?number,
 
   /**
-   * TV next focus up (see documentation for the View component).
-   *
-   * @platform android
+    Designates the next view to receive focus when the user navigates up. See
```

Comment:
Do you mean the doc style change? This code is a forked bit from Facebook's source, and we don't do the same doc generation that we do right now, so I'm not sure it immediately concerns us.

Started a different thread related to doc generation though, because we've been wanting to do something better about it for a while.

Reply:
It is the style yes. I thought we wanted to keep all coding styles matching FB's afaik the editors I've used always auto add the `*` in front after you hit enter after `/**`